src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    void returnInfo();
```

Comment:
The method 'returnInfo' should have a return type if it is meant to return information. If it's supposed to print information, consider renaming it to reflect that action, like 'printInfo'.